crates/protocol/registry/src/chain_list.rs

```diff
@@ -4,11 +4,42 @@ use alloc::{string::String, vec::Vec};
 
 /// List of Chains.
 #[derive(Debug, Clone, Default, Hash, Eq, PartialEq, serde::Serialize, serde::Deserialize)]
+#[serde(transparent)]
 pub struct ChainList {
     /// List of Chains.
     pub chains: Vec<Chain>,
 }
 
+impl ChainList {
```

Comment:
I think what could be nice here is `alloy_chains::Chain` support then you could look look via `Chain::optimism()` for example

Reply:
Yeah neat idea. Added.